.ci/build/build_frontend

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# This script builds out frontend assets for the atomic theme and
+# component library if the pull request branch exists in those repos.
+
+# Define path to atomic theme for reuse.
+ATOMIC_PATH="web/themes/contrib/atomic"
+
+# Build atomic theme. We must do this from the profile by updating composer.json,
+# removing the existing atomic directory, and running composer update.
+cd "$GITHUB_WORKSPACE"/web/profiles/custom/yalesites_profile
+
+# Require the branch version if it exists.
+if [ -n "$(git ls-remote --heads https://github.com/yalesites-org/atomic.git "$BRANCH")" ]; then
+  composer require --prefer-dist --no-update yalesites-org/atomic:dev-"$BRANCH"
+fi
+
+cd "$GITHUB_WORKSPACE"
+rm -rf "${GITHUB_WORKSPACE:?}"/"$ATOMIC_PATH"
+composer update --prefer-dist
```

Comment:
I believe all of this can be done in the if conditional block? If the branch does not exist, then all of the files should be in the right place.